$DragonFly: src/usr.bin/ftp/Attic/fetch.c.patch,v 1.1 2005/11/13 11:59:03 corecode Exp $
===================================================================
RCS file: /cvs/src/contrib/lukemftp/fetch.c,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 fetch.c
--- fetch.c	13 Nov 2005 11:28:53 -0000	1.1.1.1
+++ fetch.c	13 Nov 2005 11:30:19 -0000
@@ -65,6 +65,7 @@
 #include <netdb.h>
 #include <fcntl.h>
 #include <stdio.h>
+#include <libutil.h>
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
